src/components/Header.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+  HeaderNavigation,
+  ALIGN,
+  StyledNavigationItem as NavigationItem,
+  StyledNavigationList as NavigationList,
+} from 'baseui/header-navigation';
+import {StyledLink as Link} from 'baseui/link';
+import { Button, SIZE } from "baseui/button";
+import { ChevronDown } from "baseui/icon";
+import { StatefulPopover, PLACEMENT } from "baseui/popover";
+import { StatefulMenu } from "baseui/menu";
+import useNetwork from '../hooks/useNetwork';
+import { WalletAdapterNetwork } from '@solana/wallet-adapter-base';
+
+function Header() {
+  const {network, setNetwork} = useNetwork();
+  return (
+    <HeaderNavigation>
+      <NavigationList $align={ALIGN.left}>
+        <NavigationItem>Raindrop Protocol: Creator</NavigationItem>
+      </NavigationList>
+      <NavigationList $align={ALIGN.center}>
+        <NavigationItem>
+          <Link href="/">Create Item Class</Link>
+        </NavigationItem>
+        <NavigationItem>
+          <Link href="https://long-banana.gitbook.io/raindrops/">
+            Documentation
+          </Link>
+        </NavigationItem>
+      </NavigationList>
+      <NavigationList $align={ALIGN.right}>
+        <NavigationItem $style={{ marginRight: "40px" }}>
+          <NetworkDropdown selectedNetwork={network} setNetwork={setNetwork} />
+        </NavigationItem>
+      </NavigationList>
+    </HeaderNavigation>
+  );
+}
+
+function NetworkDropdown({selectedNetwork, setNetwork}: {selectedNetwork: WalletAdapterNetwork, setNetwork?: (arg: WalletAdapterNetwork) => void;}) {
+    const button = (
+      <Button size={SIZE.compact} endEnhancer={<ChevronDown size={24} />}>{selectedNetwork}</Button>
+    );
+    return setNetwork ? (
+      <StatefulPopover
+        focusLock
+        placement={PLACEMENT.bottomLeft}
+        content={({ close }) => (
+          <StatefulMenu
+            items={[
```

Comment:
Add localnet too please. I'd love to be able to use this for local development too.

Reply:
This ended up being a little more involved due to the lack of localnet on the wallet adapter network options - I've switched it over to sol-kit's cluster list https://github.com/raindrop-studios/creator-ui/pull/6/commits/de5273dfe298f6671fbdd6143eb80e8d26ca03db